BaseUpload.php

```diff
@@ -38,6 +39,7 @@ class BaseUpload extends InputWidget
     public $clientEvents = [];
 
 
+    public $i18n = [];
```

Comment:
There are no phpDocs + it should have no space up and there should be two spaces between this attribute and the following function